storage/src/main/java/pl/edu/icm/unity/store/export/update/UpdateFrom2_4_x.java

```diff
@@ -21,9 +23,11 @@
 
 import pl.edu.icm.unity.store.export.Update;
 import pl.edu.icm.unity.store.objstore.msgtemplate.MessageTemplateHandler;
+import pl.edu.icm.unity.store.objstore.notify.NotificationChannelHandler;
 import pl.edu.icm.unity.store.objstore.reg.eform.EnquiryFormHandler;
 import pl.edu.icm.unity.store.objstore.reg.form.RegistrationFormHandler;
 import pl.edu.icm.unity.store.objstore.reg.invite.InvitationHandler;
+import pl.edu.icm.unity.types.confirmation.EmailConfirmationConfiguration;
 
 /**
  * Update db from 2.4.1 to 2.5.0
```

Comment:
Couple of comments here. DumpSchemaVersion has new version which is wrongly defined. Old version name is wrong and the enum value is missnamed: V_INITIAL2_4_x. Something like V_2_5 ? Also in implementation I think you assumed that only a single update can be run. Note that those can be stacked (first update to 2.0 schema (no 3) is performed then to 2.5 schema (no 4)).